src/containers/Header.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link, Router, IndexRoute, Route, browserHistory } from 'react-router';
+
+
+export default class Header extends Component {
+
+  handleLocation() {
+    const geolocation = require('geolocation');
```

Comment:
How come we're requiring `geolocation` inside of a method? This will cause it to be re-required every time. Also, what's `geolocation`? Unless I'm missing it, you don't see to be using it anywhere else.